test/http/body_writer.cpp

```diff
@@ -0,0 +1,9 @@
+//
+// Copyright (c) 2013-2016 Vinnie Falco (vinnie dot falco at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+
+// Test that header file is self-contained.
+#include <beast/http/body_type.hpp>
```

Comment:
This file can be deleted. It is a duplicate of `test/http/body_type.cpp`, but does not appear to be referenced anywhere.